boot/bootstate20.go

```diff
@@ -93,6 +96,9 @@ type bootStateUpdate20 struct {
 
 	// tasks to run after the modeenv has been written
 	postModeenvTasks []bootCommitTask
+
+	// device
+	dev Device
```

Comment:
I think we should store just the model atm (when we will have to support remodeling it will get more complex but let's do things one at a time), something like resealModel

Reply:
This is done in #9340